### DISCUSSION TITLE
Add snap-up statuses on the sealing page

### DISCUSSION THREAD

**Discussion by SBudo** [filecoin-project/boost#410] [Ideas]
I noticed that, when a deal made through boost uses snap-up, the sealing UI page does not display the sealing pipeline for the sector.
Can we add this?

Same thing with the "WaitDeals" state as there is no "SnapDealsWaitDeals" state and the progress bar at the top of the page remains at zero.

![image](https://user-images.githubusercontent.com/16908232/161972637-54b43c8d-5d6d-464d-b7c5-a7c5467c1687.png)

![image](https://user-images.githubusercontent.com/16908232/161973118-b77620c0-90b2-4198-b29b-16e08e83e395.png)

**Comment by LexLuthr** [comment]
@SBudo This is already in the works. Lotus team and boost team have planned it out and we will be adding snap-deals UI in some time.

**Comment by jacobheun** [comment]
Here's the issue for reference & tracking https://github.com/filecoin-project/boost/issues/266
